src/InputContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { updateRestaurantInfo, addRestaurant } from './actions';
+import RestaurantInput from './RestaurantInput';
```

Comment:
각 모듈의 역할에 따라 공백을 통해 구분 지어주면 좋을 것 같아요~ 

```suggestion
import React from 'react';

import { useDispatch } from 'react-redux';

import { updateRestaurantInfo, addRestaurant } from './actions';

import RestaurantInput from './RestaurantInput';
```